client/app/pages/queries/VisualizationEmbed.jsx

```diff
@@ -120,8 +121,7 @@ function VisualizationEmbedFooter({
         <span className="hidden-print">
           <Tooltip title="Open in Redash">
             <Link.Button className="icon-button" href={queryUrl} target="_blank">
```

Comment:
Just a note: When we have a design system, I don't think we'll want link buttons with link icons -- I think we'll want to standardize this. No action for now.